src/components/transactions.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import '../styles/transactions.css'
+
+export default class Transactions extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            inflow: 1000,
+            outflow: 500,
+            transactions: [
+                {
+                    id: 1,
+                    spent: true,
+                    date: "01.01.20",
+                    category: "Food",
+                    amount: 100,
+                },
+                {
+                    id: 2,
+                    spent: false,
+                    date: "02.01.20",
+                    category: "Salary",
+                    amount: 1000,
+                },
+                {
+                    id: 3,
+                    spent: true,
+                    date: "03.01.20",
+                    category: "Car",
+                    amount: 350,
+                }
+            ]
+        };
+    }
+
+    calcInflow(transactions) {
+        var inflow = 0;
+        for (var transaction of transactions) {
+            if (!transaction.spent) {
+                inflow += transaction.amount
+            }
+        }
+        return inflow
+    }
+
+    calcOutflow(transactions) {
+        var inflow = 0;
+        for (var transaction of transactions) {
+            if (transaction.spent) {
+                inflow += transaction.amount
+            }
+        }
+        return inflow
+    }
+
+    openForm() {
+        document.getElementById("add-transaction").style.display = "block";
```

Comment:
this is bad. Use dynamic styles that react present.

Reply:
Removed it in the next homework, changed the way form works.